client/payments/data/selectors.js

```diff
@@ -0,0 +1,10 @@
+export function retrievePaymentGateway( state, id ) {
+	const paymentGateway = state.paymentGateways.filter( ( gateway ) => {
+		return gateway.id === id;
+	} );
+	return paymentGateway ? paymentGateway[ 0 ] : null;
+}
+
+export function listPaymentGateways( state ) {
```

Comment:
for convention sake I think `getPaymentGateways` makes more sense.

Reply:
sounds good. I tried to use the same name as the REST API endpoint but looks like I should follow the convention.